queries.sql

```diff
@@ -8,3 +8,66 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name NOT IN ('Gabumon');
 SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <=17.3;
+
+
+BEGIN;
+
+UPDATE animals
+SET species = 'unspecified';
+
+ROLLBACK;
+
+SELECT * FROM animals;
+
+
+
```

Comment:
👍 Great progress on your project! To further enhance it, here's a recommendation:

- [ ] Consider optimizing your code by removing unnecessary multi-line spaces. Excessive spacing can make your files larger and less readable. Aim for consistent and concise code formatting with single-line spaces between code blocks and sections.

By following this suggestion, you can improve the readability and maintainability of your codebase. Keep up the good work! 👍🧹📜

Reply:
Thanks for the suggestion